uber/templates/base.html

```diff
@@ -93,7 +93,15 @@
             if (message.length) {
                 toastr.info(message);
             }
-            $('.datatable').DataTable();
+            if ($('.datatable').length) {
+                var datatable = $('.datatable').dataTable({
+                    aLengthMenu: [
+                        [25, 50, 100, 200, -1],
+                        [25, 50, 100, 200, "All"]
+                        ],
+                    stateSave: true
+                    });
+            }
```

Comment:
Two things:

1) You don't need the ``.length`` if-check because it's perfectly fine to call ``.DataTable()`` on an empty jQuery list.  This is a general convention for jQuery actions; saying ``.FooBar()`` means to apply ``FooBar`` to everything (or occasionally only the first thing) in the selection, but no items means it's a noop.

2) Super-nitpicky, but your closing ``});`` is indented one level (4 spaces) too far.